### PR TITLE
Add JetBrains Exposed dependencies

### LINKED ISSUE #417: Dependency notations for JetBrains/Exposed
## About the library(-ies)

- *Description*: Kotlin SQL Framework 
- *GitHub*: https://github.com/JetBrains/Exposed
- *Official website*: https://github.com/JetBrains/Exposed/wiki/Getting-Started#download

## Wants to contribute the sample?

- Say so in a comment here
- Follow [instructions for submitting dependency notations](https://jmfayard.github.io/refreshVersions/contributing/submitting-prs/dependency-notations-updates/)
- Submit your pull request!

### PR DESCRIPTION
**Pull request by imashnake0**
Hello/bonjour, I've added JetBrains Exposed dependencies. 

## What?

Resolves #417.

## Why?

To resolve #417.

## How?

Added dependencies according to #417.

## Testing?

* Verified additions to `versions.properites` in sample-android.
* Ran `./gradlew check`.

### CODE REVIEW

File: plugins/dependencies/src/main/resources/refreshVersions-rules/jetbrains-version-alias-rules.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+org.jetbrains.exposed:exposed-*
+    ^^^^^^^^^.^^^^^^^
```
Comment (jmfayard) [inline plugins/dependencies/src/main/resources/refreshVersions-rules/jetbrains-version-alias-rules.txt:2]: Can you add the rules as parameter to DependencyGroup?
https://github.com/jmfayard/refreshVersions/blob/56fdfbb452cc7928f4fb136b67640f1f0f3ad370/plugins/dependencies/src/main/kotlin/dependencies/Kodein.kt#L17-L23


Same for your other pull requests

**Comment by jmfayard** [comment]
Thanks @imashnake0 
Any reason why it's a draft pull request?

**Comment by imashnake0** [comment]
> Thanks @imashnake0 Any reason why it's a draft pull request?

Figured I'd thoroughly go through my changes one last time. I'll mark them as Ready for review later today.

**Comment by LouisCAD** [comment]
We have a duplicated rule in this PR now @jmfayard

File: plugins/dependencies/src/main/resources/refreshVersions-rules/jetbrains-version-alias-rules.txt (hunk 1 above)
Comment (LouisCAD) [reply plugins/dependencies/src/main/resources/refreshVersions-rules/jetbrains-version-alias-rules.txt:2]: We have a duplicated rule in this PR now @jmfayard

I think we should either keep it like it was for dependencies in the JetBrains group, or move them all, but I don't think we really want that, and we definitely don't want that in this PR.

**Comment by LouisCAD** [comment]
Converted back to a draft so we discuss the concern I raised about the duplicated rule and what option we will choose.

**Comment by jmfayard** [comment]
Merged via https://github.com/jmfayard/refreshVersions/pull/559

Thanks a lot @imashnake0 

Sorry it took so long, I was busy with life

**Pull request closed without merging**